drivers/bluetooth/hci/Kconfig.infineon

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2023 Cypress Semiconductor Corporation (an Infineon company) or
+# an affiliate of Cypress Semiconductor Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# Change size of command lengths. It for vendor commands related to
+# firmware downloading.
+config BT_BUF_CMD_TX_SIZE
+	default 255
```

Comment:
I ran this on a board with a 1DX and also had to set

```
config BT_ATT_ENFORCE_FLOW
       default n

config BT_BUF_CMD_TX_COUNT
       default 10
```

the first was after some warning message about commands out of order, the second was for some more insidious buffer overflow, hopefully it'll save you from troubleshooting it as well

Reply:
Yeah I think it makes more sense to have them somewhere in this file, probably here if they apply to the whole family.